src/util/Permissions.js

```diff
@@ -112,6 +126,15 @@ class Permissions {
     if (typeof permission === 'string') return this.FLAGS[permission];
     throw new RangeError('PERMISSIONS_INVALID');
   }
+
+  /**
+   * Resolves permissions to an instance of {@link Permissions}
+   * @param {PermissionResolvable|PermissionResolvable[]} permission - Permission(s) to resolve
+   * @returns {Permissions}
+   */
+  static resolveToObject(permission) {
```

Comment:
Useless: the entirety of this method is handled in the constructor.

Reply:
ik, check my latest comment